lib/matplotlib/axes/_axes.py

```diff
@@ -4583,14 +4572,9 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
 
         Other Parameters
         ----------------
-        cmap : str or `~matplotlib.colors.Colormap`, default: :rc:`image.cmap`
-            The Colormap instance or registered colormap name used to map
-            the bin values to colors.
-
-        norm : `~matplotlib.colors.Normalize`, optional
-            The Normalize instance scales the bin values to the canonical
-            colormap range [0, 1] for mapping to colors. By default, the data
-            range is mapped to the colorbar range using linear scaling.
+        cmap, norm, vmin, vmax
+            The data normalization and colormapping, which are explained in
+            detail in the documentation for `~matplotlib.axes.Axes.imshow`.
```

Comment:
```suggestion
            Data normalization and colormapping parameters. See
            `~matplotlib.axes.Axes.imshow` for a detailed description.
```
I can't exactly tell why, but this wording sounds slightly better to me.